setup.py

```diff
@@ -35,10 +35,18 @@ def read(fname):
       author_email='matti.pastell@helsinki.fi',
       url='http://mpastell.com/pweave',
       packages=['pweave', 'pweave.themes', 'pweave.formatters', 'pweave.processors', 'pweave.bokeh'],
-      install_requires = ['markdown', 'pygments', 'ipython', 'nbformat',
-                          'nbconvert', 'jupyter_client', 'ipykernel'],
-      extras_require = {'test': ['scipy', 'matplotlib', 'coverage',
-                                   'ipython', 'nose', 'notebook'],
+      install_requires = ['markdown',
+                          'pygments',
+                          'ipython<=6.2.0',
+                          'nbformat',
+                          'nbconvert',
+                          'jupyter_client',
+                          'ipykernel'],
+      extras_require = {'test': ['scipy',
+                                 'matplotlib',
+                                 'coverage',
+                                 'nose',
```

Comment:
'ipython' removed. Explanation necessary in the commit description.

Reply:
if ipython is in the install_requires it is redundant to list in the extras_require

Reply:
That would be an excellent explanation in the commit comment. Even "redundant dependency removed" will do if the commit is restricted to that change.

Reply:
I'm not sure what you are asking for here. Are you requiring me to amend the commit message on the particular relevant commit, or can this be in the commit message for the merge commit.

Reply:
@fgregg if amendment is a problem, it may be fixed in the merge comment.